fidesctl/src/fideslang/models.py

```diff
@@ -204,6 +207,7 @@ class Organization(FidesModel):
     controller: Optional[ContactDetails]
     data_protection_officer: Optional[ContactDetails]
     representative: Optional[ContactDetails]
+    security_policy: Optional[str]
```

Comment:
should this actually be a URL? pydantic supports types like that

Reply:
I'm down with that 👍🏽  I wasn't sure how strict we wanted to be with it initially but I think based on [conversations in the original issue](https://github.com/ethyca/fides/issues/322#issuecomment-1017941009) it makes sense to be more explicit about what is expected here, will get that updated

Reply:
Updated and tested using `HttpUrl` - I may need to go back in to the doc updates here and clarify the need for a proper URL in case of issues where a malformed url is used.

Reply:
it _should_ gracefully throw an error in that case, pydantic will check it